transcendence-app/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Controller,
+  Get,
+  Request as GetRequest,
+  UseGuards,
+} from '@nestjs/common';
+import {
+  ApiBadGatewayResponse,
+  ApiFoundResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+import { Request } from 'express';
+import { OAuth42Guard } from './oauth42.guard';
+
+@ApiTags('auth')
+@Controller('auth')
+export class AuthController {
+  @Get('oauth42')
```

Comment:
Suggestion: There is a 'logout' endpoint, what if we call 'oauth42' endpoint 'login'?